base/libgit2/libgit2.jl

```diff
@@ -71,22 +71,34 @@ function iscommit(id::AbstractString, repo::GitRepo)
     return res
 end
 
-""" git diff-index HEAD [-- <path>]"""
+"""
+    LibGit2.isdirty(repo::GitRepo[, paths]; cached=false)
+
+Checks if there have been any changes to tracked files in the working tree (if
+`cached=false`) or the index (if `cached=true`).
+
+See `git diff-index HEAD [-- <path>]`
+"""
 isdirty(repo::GitRepo, paths::AbstractString=""; cached::Bool=false) =
     isdiff(repo, Consts.HEAD_FILE, paths, cached=cached)
 
-""" git diff-index <treeish> [-- <path>]"""
+"""
+    LibGit2.isdiff(repo::GitRepo, treeish[, paths]; cached=false)
```

Comment:
not something that this pr should address, but "isdiff" is an odd name - sounds like it's asking whether something is a diff